solution/src/sum-evens.js

```diff
@@ -0,0 +1,8 @@
+function sumEvens(numbers) {
+    if (!Array.isArray(numbers)) {
+        return undefined;
+    }
+    return numbers.filter((x) => x % 2 == 0).reduce((x, y) => x + y, 0);
```

Comment:
:100:

Reply:
I've renamed to:
```javascript
return numbers.filter((number) => number % 2 == 0)
        .reduce((previousValue, currentValue) => previousValue + currentValue, 0);
```
Perhaps a bit better. But I can't illustrate to the reader the entire logic of the `reduce` function with just the parameter names. 
`x` and `y` are a kind of iterators here, similarly to the `i` in `for(i=0; i<n; i++)`. You wouldn't want me to rename the `i` to something like `aValueThatGetsIncrementedWithEachIterationOfTheForLoop`, right?

It could be just me. Let me know of better parameter names for `reduce`, please. Thank you!